app/javascript/.storybook/main.js

```diff
@@ -72,6 +73,7 @@ module.exports = {
         ...config.resolve.alias,
         '@crayons': path.resolve(__dirname, '../crayons'),
         '@utilities': path.resolve(__dirname, '../utilities'),
+        '@img': path.resolve(__dirname, '../../assets/images'),
```

Comment:
I've added this alias so it should be easier to import icons:

```
import Icon from '@img/icon.svg';

instead of 

import Icon from '../../../../../../../whatever/whatever/images/icon.svg';
```
```

Reply:
LOVE